scripts/001-binutils.sh

```diff
@@ -27,8 +27,13 @@ rm -rf build-$TARGET && mkdir build-$TARGET && cd build-$TARGET || { exit 1; }
   --quiet \
   --prefix="$PS2DEV/$TARGET_ALIAS" \
   --target="$TARGET" \
+  --enable-interwork \
+  --enable-plugins \
```

Comment:
Why are these 2 enabled in a commit that only disables things?

Reply:
Removed it.